themes/classic/templates/checkout/_partials/steps/payment.tpl

```diff
@@ -121,6 +121,8 @@
     </form>
   {/if}
 
+  {hook h='displayCheckoutBeforeConfirmation'}
```

Comment:
Maybe it's better to have a name likse displayBeforeCheckoutConfirmation, in order to ensure the logic user in another part like displayBeforeCarrier ?

Reply:
other hooks are like `displayCheckoutSummaryTop` `displayCheckoutSomething`, etc.

It doesn't sound good, but it makes sense when it comes to searching for the right hook.

Reply:
You're right, about them ! :)